client/src/pages/VotePage.tsx

```diff
@@ -59,6 +65,13 @@ export const VotePage = () => {
         setSelectionOption("");
         if (!isFocus) {
           document.title = questionStarted;
+          if (notifPermission === "granted") {
```

Comment:
At least from testing this, it doesn't look like a notification will trigger for every new question (tested this on Windows 11). It seems like it'll trigger once, the user would have to either click on the notification to acknowledge it or to clear their notifications for it to retrigger. Just dismissing the notification doesn't seem to make it trigger anymore.

What's the functionality that you're trying to achieve for this? (For a notification on every single new question when out of focus?)

Reply:
> it doesn't look like a notification will trigger for every new question (tested this on Windows 11). It seems like it'll trigger once, then the user would have to either click on the notification to acknowledge it or to clear their notifications for it to retrigger. Just dismissing the notification doesn't seem to make it trigger anymore.

I fired up a VM and was also able to reproduce this behavior. It should be fixed with the latest commit.

> What's the functionality that you're trying to achieve for this? (For a notification on every single new question when out of focus?)

Yes, exactly. Currently it seems that only the title is updated when a new question starts, which means the user needs to be looking at the browser to be aware that they should vote. I think a notification is a cleaner solution and is probably better for accessibility purposes too (since a screen reader could announce it).

Notably, if a user does not want to see the notifications, they can simply turn off the permission altogether in the URL bar.

![image](https://user-images.githubusercontent.com/42941056/194755610-5c8f5ee4-1e5c-45f2-9006-5f5bebb47ab7.png)